examples/webpack.config.js

```diff
@@ -1,7 +1,7 @@
 var path = require('path');
 var webpack = require('webpack');
 var webpackBase = require('./webpack.base');
-var assign = require('lodash/object').assign;
+var assign = require('lodash.assign');
```

Comment:
I don't neither, I'm already refactoring these files in pure ES6 in a different branch (so `...` instead of `assign`) :)